### ISSUE TITLE
discovery and account tab not working properly

### ISSUE THREAD

**Issue by Ivan2023** [goatfungus/NMSSaveEditor#814]
I'm on windows using the jar file on a steam save. Whenever i click on either tabs nothing happens. It is not until i move my mouse over where the buttons, checkboxes and sliders are that things start to pop up. but not everything appears as text outside the scroll menus stays invisible, you are still able to see content of the previous tab, and it is nearly impossible to use those tabs as you cant see any buttons and don't know what you're clicking on. Clicking to a different tab works fine so it doesn't break the editor. Also a small side request is if you could add a save transfer between different platforms as editing the json files isn't something a lot of people are comfortable with as a lot could go wrong

**Comment by GrumpyKoala1966** [comment]
I've just updated as well and everything works as intended from my side..

**Comment by goatfungus** [comment]
Duplicate of #811
If you could add further information to the other issue, such as your log file, that would be helpful.
